aws_marketplace/curating_aws_marketplace_listing_and_sample_notebook/Algorithm/curating_good_algorithm_listing.md

```diff
@@ -0,0 +1,46 @@
+| PO1 | Product overview | Describe algorithm category e.g., Tree, Neural Net etc. Ensemble                                                        | Mandatory                    | An AutoML algorithm that trains a multi-layer stack ensemble model to predict on regression/classification datasets directly from CSV data.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
+| PO2 | Product overview | Summarize how the algorithm works including any feature engineering                                                     | Mandatory                    | AutoGluon-Tabular can save time by automating time-consuming manual steps—handling missing data, manual feature transformations, data splitting, model selection, algorithm selection, hyperparameter selection and tuning, ensembling multiple models, and repeating the process when data changes                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
+| H4  | Highlights       | Specify if algorithm is compatible with model auto tuning functionality, distributed training, GPU on Amazon SageMaker. | Mandatory                    | The model can be trained using automatic model tuning capability and you can specify multiple instances while running a model training job                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
```

Comment:
This part is a little unclear to me. Do you mean a list of possibilities?
```suggestion
| H4  | Highlights       | Specify if algorithm is compatible with, for example, model auto tuning functionality, distributed training, GPU on Amazon SageMaker. | Mandatory                    | The model can be trained using automatic model tuning capability and you can specify multiple instances while running a model training job                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
```

Reply:
yes, we want sellers to specify three things: 1. does it support auto tuning functionality 2. distributed training 3. 
Training with GPUs